orchestra/static/orchestra/common/js/orchestra.services.es6.js

```diff
@@ -72,6 +72,7 @@ export function orchestraTasks ($http) {
   const pendingState = (task) => ['pending_review', 'pending_processing'].indexOf(task.state) !== -1
   const activeTask = (task) => activeState(task) && task.should_be_active
   const pendingTask = (task) => (activeState(task) && !task.should_be_active) || pendingState(task)
+  const pausedTask = (task) => task.state === 'paused'
```

Comment:
'paused' or 'Paused'?

Reply:
'paused' for task state (we used 'Paused' for project).